src/ts/src/common/utils/urlUtils.tsx

```diff
@@ -0,0 +1,7 @@
+export function stripProtocol(url: string | undefined): string | undefined {
+  if (url === undefined) {
+    return undefined;
+  }
+  const re = /^.*:\/\//i;
```

Comment:
do we need to strip out other protocols here besides just https?

Reply:
This would also work on `http://`. It was pretty quick and easy to generalize

Reply:
i was just thinking that we could simplify this re to http?s or something like that to make it more obvious as to what this is stripping out.